communication/src/test/java/org/jboss/da/communication/aprox/AproxConnectorTest.java

```diff
@@ -40,13 +40,16 @@
     @Mock
     private Logger log;
 
+    @Spy
+    private MetadataFileParser parser = new MetadataFileParser();
```

Comment:
Why adding @Spy and also initializing the entity?

Reply:
@Spy can be used only on initialized fields. The @Spy was put here so the @InjectMocks would recognize it and inject it, but since it's not injected anymore I forgot to remove it from here too.

Reply:
I'm OK with uninitialized fields as well.